waku/v2/node/wakunode2.nim

```diff
@@ -573,11 +584,40 @@ proc mountRelay*(node: WakuNode,
 
   when defined(rln):
     if rlnRelayEnabled:
-      # TODO pass rln relay inputs to this proc, right now it uses default values that are set in the mountRlnRelay proc
+      # TODO get user inputs via cli options
       info "WakuRLNRelay is enabled"
-      waitFor mountRlnRelay(node)
-      info "WakuRLNRelay is mounted successfully"
 
+      # a static list of 50 membership keys in hexadecimal format
+      var groupKeys = StaticGroupKeys
+      var groupSize = int(groupKeys.len/2)
```

Comment:
For interest's sake, any specific reason these are `var` and not `let`? I may be missing it, but I don't think these variables are modified after construction in which case `let` will be more appropriate. See [style guide](https://status-im.github.io/nim-style-guide/language.vardecl.html#variable-declaration-languagevardecl)

Reply:
Right, nice catch! forgot to take them back to `let`  :),  I defined them as `var` since I needed to update their content afterward, later I have changed the logic but forgot to edit this part. Thanks for the comment!

Reply:
done in https://github.com/status-im/nim-waku/commit/925f5863488363761526a874dfd54c9aac65d275